File: app/src/main/java/rod/bailey/trafficatnsw/cameras/filter/AdmitFavouritesTrafficCameraFilter.java
package rod.bailey.trafficatnsw.cameras.filter;

import rod.bailey.trafficatnsw.cameras.TrafficCamera;

public class AdmitFavouritesTrafficCameraFilter implements ITrafficCameraFilter {

	@Override
	public boolean admit(TrafficCamera camera) {
		assert camera != null;
		return camera.isFavourite();
	}

}

File: app/src/main/java/rod/bailey/trafficatnsw/cameras/filter/AdmitFavouritesTrafficCameraFilter.kt
package rod.bailey.trafficatnsw.cameras.filter

import rod.bailey.trafficatnsw.cameras.data.XCamera

class AdmitFavouritesTrafficCameraFilter : ITrafficCameraFilter {
	override fun admit(camera: XCamera): Boolean {
		return camera.favourite
	}
}
